src/devhub/page/community/index.jsx

```diff
@@ -300,6 +301,19 @@ return (
             </Link>
           </small>
         </div>
+        <Widget
```

Comment:
To fix the test, can you do:

{context.accountId && <Widget .... }

We only show post button if user is logged in